pkg/server/api/agent/v1/service_test.go

```diff
@@ -1970,27 +1970,22 @@ func (s *serviceTest) setupNodes(ctx context.Context, t *testing.T) {
 }
 
 func (s *serviceTest) setupJoinTokens(ctx context.Context, t *testing.T) {
-	_, err := s.ds.CreateJoinToken(ctx, &datastore.CreateJoinTokenRequest{
-		JoinToken: &datastore.JoinToken{
-			Token:  "test_token",
-			Expiry: time.Now().Unix() + int64(60*10),
-		},
+	now := time.Now().Truncate(time.Second)
+	err := s.ds.CreateJoinToken(ctx, &datastore.JoinToken{
+		Token:  "test_token",
+		Expiry: now.Add(time.Second * 600),
```

Comment:
This was a pre-existing problem, but these times should be created using the mock clock (which should be stored on the serviceTest, but isn't).